libs/service/api_service.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ *  Copyright (c) Microsoft Corporation
+ *  SPDX-License-Identifier: MIT
+ */
+
+#include <map>
+#include <stdexcept>
+#include "api_common.hpp"
+#include "map_descriptors.hpp"
+#include "ebpf_api.h"
+#include "api_internal.h"
+#include "ebpf_execution_context.h"
+#include "ebpf_protocol.h"
+#include "ebpf_platform.h"
+#include "pch.h"
+#include "platform.h"
+extern "C"
+{
+#include "ubpf.h"
+}
```

Comment:
I believe this file should include api_service.h so you get the prototype for ebpf_verify_program, and then you can remove any includes that are included inside that header

Reply:
done.